### PR TITLE
Simple Ping MongoDB Checker

### PR DESCRIPTION
**Pull request by helios-ag**
Attempt to solve #3 with this simple mongodb ping checker.

### CODE REVIEW

File: checkers/mongo.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
+package checkers
+
+import (
+	"fmt"
+	"github.com/globalsign/mgo"
+)
+
+type MongoConfig struct {
+	Auth       *MongoAuthConfig
```
Comment (dselans) [inline checkers/mongo.go]: Please add comments around each field, indicating whether it's required, optional and how the field will influence the checker.

Something similar to this: https://github.com/InVisionApp/go-health/blob/master/checkers/sql.go#L36-L82

File: checkers/mongo.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+	return &Mongo{
+		Config: cfg,
+		Session: session,
+	}, nil
+}
+
+func (m *Mongo) Status() (interface{}, error) {
+	if m.Config.Ping {
+		fmt.Printf("Checking ping")
```
Comment (dselans) [inline checkers/mongo.go]: Remove debug statement.

File: checkers/mongo_test.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+		Expect(err).To(HaveOccurred())
+		Expect(err.Error()).To(ContainSubstring("At minimum, either cfg.Ping or cfg.Collection"))
+	})
+
+}
+
+func TestMongoStatus(t *testing.T) {
+	RegisterTestingT(t)
+
+	t.Run("Shouldn't error when ping is enabled", func(t *testing.T) {
```
Comment (dselans) [inline checkers/mongo_test.go]: Please add test case for when checker has `.Collection` set.

File: checkers/mongo_test.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+		}
+
+		r, err := NewMongo(cfg)
+		Expect(err).To(HaveOccurred())
+		Expect(err.Error()).To(ContainSubstring("no reachable servers"))
+		Expect(r).To(BeNil())
+	})
+}
+
+func TestValidateMongoConfig(t *testing.T) {
```
Comment (dselans) [inline checkers/mongo_test.go]: Please add test case for invalid auth url as per: https://codecov.io/gh/InVisionApp/go-health/pull/51/diff?src=pr&el=tree#D3-93

**Review by dselans — CHANGES_REQUESTED** [review]
Hi, thank you very much for your contribution. I think a simple `ping` check is just what we needed; the addition of `.Collection` checking is a nice bonus.

I've added a few minor comments/requests before we get this merged.

File: checkers/mongo.go (hunk 1 above)
Comment (helios-ag) [reply checkers/mongo.go]: Description added, redis checker used as template

File: checkers/mongo_test.go (hunk 4 above)
Comment (helios-ag) [reply checkers/mongo_test.go]: done

File: checkers/mongo_test.go (hunk 3 above)
Comment (helios-ag) [reply checkers/mongo_test.go]: Done

**Review by dselans — APPROVED** [review]
Thank you very much for the contrib (and requested changes)!

**Pull request merged**